### ISSUE TITLE
ResourceHeader cells should be navigatable via keys and fire keyboard events

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#981]
https://www.bryntum.com/forum/viewtopic.php?f=51&t=14406&p=73714

Currently only mouse events are fired
```
onResourceMouseEvent(event) {
        const
            resourceCell = event.target.closest('.b-resourceheader-cell'),
            resourceRecord     = this.resourceStore.getById(resourceCell.dataset.resourceId);

        this.trigger('resourceHeader' + StringHelper.capitalizeFirstLetter(event.type), {
            resourceRecord,
            event
        });
    }
```
